blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages.properties

```diff
@@ -0,0 +1,12 @@
+bo.dashboard.pipelines.heading=Dashboard
+bo.dashboard.pipelines.new.pipeline=New Pipeline
+bo.dashboard.pipelines.table.health=Health
+bo.dashboard.pipelines.table.branches=Branches
+bo.dashboard.pipelines.table.pr=Pull Requests
+bo.dashboard.loading=Loading...
+bo.dashboard.more=Show more
+bo.dashboard.failing={0} failing
```

Comment:
Must it be `{0}`, or can it be `{jobName}` (or whatever) ? I really do not think these need to be usable on the server side.